### DISCUSSION TITLE
Icon request: IRC

### DISCUSSION THREAD

**Discussion by ysim** [FortAwesome/Font-Awesome#21117] [Icon Requests]

**Comment by TheSisb** [comment]
Does IRC have an icon?  iirc it is distinctive by its lack of images.  IRC (text) is pretty clear.

**Comment by gandalfx** [comment]
I support this. I'm using the `#` character instead, which doesn't really fit with the other icons in a row of contact options.

Also some weird people will probably think that `fa-hashtag` would be a cool idea.

**Comment by RileyStarlight** [comment]
hashtag icon: #1123 
but fa-irc would be nice

**Comment by VA6DAH** [comment]
+1

**Comment by k0nsl** [comment]
I'm also all for this idea. Please? :)

**Comment by cosenal** [comment]
+1

**Comment by verschoof** [comment]
:+1:

**Comment by ramondelafuente** [comment]
:+1:

**Comment by PeterMain** [comment]
+1

**Comment by greenbigfrog** [comment]
+1
IRC simply written in a circle would also be fine...

![image](https://cloud.githubusercontent.com/assets/6208894/5509179/b296b308-87b5-11e4-9b6f-2e7fb5d9a490.png)

(This is just a example! Don't use this!)

**Comment by manuwell** [comment]
+1

**Comment by alick** [comment]
:+1:

**Comment by macskay** [comment]
+1

**Comment by anands** [comment]
+1

**Comment by HeroCC** [comment]
Support!

**Comment by arnaudoff** [comment]
+1

**Comment by suunkie** [comment]
+1

**Comment by iakat** [comment]
+1

**Comment by tronsha** [comment]
+1

**Comment by nicktdrizzy** [comment]
+1

**Comment by k0nsl** [comment]
Any progress on this? I could have sworn that I've seen an entry for IRC. But now I cannot find it.

**Comment by JeroenDeDauw** [comment]
+1

**Comment by BeauBouchard** [comment]
+1

**Comment by jwinterm** [comment]
:+1:

**Comment by j-degreef** [comment]
+1

**Comment by stmcginnis** [comment]
+1

**Comment by JoseALermaIII** [comment]
+1

Glad to see it on the [Icon Leaderboard](https://fontawesome.com/community/leaderboard/new)

**Comment by abrantesasf** [comment]
+1

**Comment by thomascorthals** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Released in 7.2: https://fontawesome.com/icons/rectangle-irc?f=classic&s=solid
